.coveragerc

```diff
@@ -597,7 +597,7 @@ omit =
     homeassistant/components/oasa_telematics/sensor.py
     homeassistant/components/ohmconnect/sensor.py
     homeassistant/components/ombi/*
-    homeassistant/components/onewire/sensor.py
+    homeassistant/components/onewire/*
```

Comment:
```suggestion
    homeassistant/components/onewire/__init__.py
```

Reply:
config flow requires 100% coverage ;)

Reply:
`pytest --cov=homeassistant/components/onewire/ --cov-report term-missing -- tests/components/onewire/test_*.py`

Reply:
I have merged in the suggested changes from @ctalkington.
Do I need to include also the __init__.py, config_flow.py and const.py?

Reply:
Coverage failed, so added those back in: init.py, config_flow.py and const.py

Reply:
@ctalkington 
I have now added config_flow tests in 91f91fe. Are you happy to mark this as resolved?